src/test/java/spoon/test/annotation/AnnotationTest.java

```diff
@@ -296,7 +297,7 @@ public void testAnnotatedElementTypes() throws Exception {
 		assertTrue(annotations.get(0).getAnnotatedElement().equals(enumeration));
 		assertEquals(CtAnnotatedElementType.TYPE, annotations.get(0).getAnnotatedElementType());
 
-		List<CtField<?>> fields = enumeration.getValues();
```

Comment:
if we are backward compatible only with deprecation, we should keep the previous getValues and assertions